Platformer2D/Platforms/Android/Platformer2D.csproj

```diff
@@ -12,20 +12,21 @@
     <RootNamespace>Platformer2D</RootNamespace>
     <AssemblyName>Platformer2D</AssemblyName>
     <FileAlignment>512</FileAlignment>
-    <AndroidApplication>true</AndroidApplication>
+    <AndroidApplication>True</AndroidApplication>
     <AndroidResgenFile>Resources\Resource.Designer.cs</AndroidResgenFile>
-    <GenerateSerializationAssemblies>Off</GenerateSerializationAssemblies>
-    <AndroidSupportedAbis>armeabi%3barmeabi-v7a%3bx86</AndroidSupportedAbis>
-    <AndroidStoreUncompressedFileExtensions />
-    <MandroidI18n />
-    <TargetFrameworkVersion>v3.1</TargetFrameworkVersion>
+    <MonoAndroidResourcePrefix>Resources</MonoAndroidResourcePrefix>
+    <MonoAndroidAssetsPrefix>Assets</MonoAndroidAssetsPrefix>
+    <TargetFrameworkVersion>v9.0</TargetFrameworkVersion>
     <AndroidManifest>Properties\AndroidManifest.xml</AndroidManifest>
+    <AndroidEnableSGenConcurrent>true</AndroidEnableSGenConcurrent>
+    <AndroidUseAapt2>true</AndroidUseAapt2>
+    <AndroidHttpClientHandlerType>Xamarin.Android.Net.AndroidClientHandler</AndroidHttpClientHandlerType>
     <JavaMaximumHeapSize />
     <JavaOptions />
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\..\</SolutionDir>
     <RestorePackages>true</RestorePackages>
-    <NuGetPackageImportStamp>f7f092e7</NuGetPackageImportStamp>
-    <MonoGamePlatform>Android</MonoGamePlatform>
```

Comment:
Are we sure this line should be removed?  This was provided as part of the MonoGame platform setup to identify the platform

Reply:
I believe the NuGetPackageImportStamp is a holdover of the pre-PackageReference world. It was used in the packages.config old style. I don't believe it is necessary anymore. In fairness, I'm not 100% sure of the significance but I tested clean builds and I know it is not strictly speaking necessary.

The MonoGamePlatform=Android specification is definitely not necessary. The reason is the .targets file in the MonoGame.Framework.Android nuget package already specifies that.